src/wp-includes/default-filters.php

```diff
@@ -195,16 +195,17 @@
 add_filter( 'the_content', 'wpautop' );
 add_filter( 'the_content', 'shortcode_unautop' );
 add_filter( 'the_content', 'prepend_attachment' );
-add_filter( 'the_content', 'wp_filter_content_tags' );
 add_filter( 'the_content', 'wp_replace_insecure_home_url' );
+add_filter( 'the_content', 'do_shortcode', 11 ); // AFTER wpautop().
+add_filter( 'the_content', 'wp_filter_content_tags', 12 ); // Runs after do_shortcode().
```

Comment:
```suggestion
add_filter( 'the_content', 'wp_filter_content_tags', PHP_INT_MAX ); // Runs after any other hooks that may inject images.
```

Reply:
It seems [some plugins](https://wpdirectory.net/search/01HB6M3FT27M1BSX4QQV25MD8R) also use `PHP_INT_MAX`. Is this likely to cause a problem?